modelChecker/modelChecker_commands.py

```diff
@@ -129,26 +136,21 @@ def zeroLengthEdges(_, SLMesh):
         selIt.next()
     return zeroLengthEdges
 
-
 def selfPenetratingUVs(transformNodes, _):
     selfPenetratingUVs = []
     for node in transformNodes:
-        shape = cmds.listRelatives(node, shapes=True, fullPath=True)
-        convertToFaces = cmds.ls(
-            cmds.polyListComponentConversion(shape, tf=True), fl=True)
-        overlapping = (cmds.polyUVOverlap(convertToFaces, oc=True))
-        if overlapping:
-            for node in overlapping:
-                selfPenetratingUVs.append(node)
+        shape = cmds.listRelatives(node, shapes=True, fullPath=True, type="mesh")
+        faces = cmds.polyListComponentConversion(shape, tf=True)
+        if overlapping := cmds.polyUVOverlap(faces, oc=True):
```

Comment:
I wonder if this also works.
```suggestion
        if overlapping := cmds.polyUVOverlap("{}.f[*]".format(shape), oc=True):
```

Reply:
I like it! I'll give it a shot. Great suggestion!